core/src/main/java/org/fao/geonet/kernel/security/shibboleth/ShibbolethUserUtils.java

```diff
@@ -95,20 +98,47 @@ protected UserDetails setupUser(ServletRequest request, ShibbolethUserConfigurat
 		String username = getHeader(req, config.getUsernameKey(), "");
 		String surname = getHeader(req, config.getSurnameKey(), "");
 		String firstname = getHeader(req, config.getFirstnameKey(), "");
+        String organisation = getHeader(req, config.getOrganisationKey(), "");
```

Comment:
Indentation

Reply:
In the `IDE` looks fine, but showing whitespace chars, the file has a mix of spaces/tabs, fixing it to replace with spaces.